markdown/usage/tutorials/nf_core_tutorial.md

```diff
@@ -8,21 +8,25 @@ subtitle: Tutorial covering the basics of using and creating nf-core pipelines.
 > Duration: **1hr 45**
 >
 > Updated for the nf-core Hackathon 2020, London 2020-03 _(see [event](https://nf-co.re/events#hackathon-francis-crick-2020))._
+>
+> Updated for the Elixir workshop on November 2021 _(see [event](https://nf-co.re/events/2021/elixir-workflow-training-event))._
 
 <!-- markdownlint-disable -->
+<!-- TODO: update slides here -->
 <iframe src="//www.slideshare.net/slideshow/embed_code/key/sToqg2FJcGUJZ2" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
 <!-- markdownlint-restore -->
 
 **[Click here to download the slides associated with this tutorial.](/assets/markdown_assets/usage/nf_core_tutorial/nf-core_tutorial.pdf)**
 
 #### Exercises
 
```

Comment:
I've updated here the index to point the corresponding sections rather than the exercises